### DISCUSSION TITLE
Console open

### DISCUSSION THREAD

**Discussion by napalmextreme** [ultrafunkamsterdam/undetected-chromedriver#1311] [General]
when I compile even in --windoed the console of chrome undetected is open , how do I hide it while the application is open

**Comment by sahariar-safin** [comment]
There are a few ways to hide the Chrome console while your application is open.

`Use the --disable-web-console flag`

This flag will disable the Chrome console altogether. To use this flag, simply add it to your command when you compile your application. For example, if you are compiling your application with the command` python my_app.py`, you would change it to `python my_app.py --disable-web-console`.

- Close the Chrome console manually

If you only want to close the Chrome console temporarily, you can do so by clicking on the "X" in the top right corner of the console window. The console will be closed, but it will reopen the next time you open your application.

- Use a Chrome extension

There are a number of Chrome extensions that can be used to hide the console. One popular extension is called "Console Hider". This extension allows you to hide the console with a single click. To install Console Hider, simply go to the Chrome Web Store and search for "Console Hider". Once you have installed the extension, you can enable it by clicking on the three dots in the top right corner of the Chrome window and selecting "Extensions". From there, you can enable the extension by clicking on the toggle switch next to it.

Whichever method you choose, make sure that you test your application thoroughly to make sure that the console is not interfering with its functionality.
